### ISSUE TITLE
Error for load library and "Unexpected token export"

### ISSUE THREAD

**Issue by marcdubois71450** [gullerya/object-observer#27]
The call of the files is not working properly.
`let Observable = require('./dist/node/object-observer').Observable;`

I had to replace it with
`let Observable = require('./node_modules/object-observer/dist/node/object-observer').Observable;`

And now that I managed to load the library I have an error

```
/node_modules/object-observer/dist/node/object-observer.js:632
export { Observable };
^^^^^^

SyntaxError: Unexpected token export
    at new Script (vm.js:79:7)
    at createScript (vm.js:251:10)
    at Object.runInThisContext (vm.js:303:10)
    at Module._compile (internal/modules/cjs/loader.js:657:28)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
```

**Comment by gullerya** [comment]
Yep, the post-processing of the node dist got broken for some silly reason, i'm fixing it.
And I'll add a notice about the usage in NodeJS (the relevant path).

**Comment by gullerya** [comment]
Fixed as part of 2.4.0, closing the issue, feel free to reopen or open another one if there is still something missing.

**Comment by marcdubois71450** [comment]
Thank you very much !
